site/in-dev/cli.md

```diff
@@ -5,35 +5,55 @@ title: "Nessie CLI"
 # Nessie CLI
 
 The Nessie CLI is an easy way to get started with Nessie. It supports multiple branch 
-and tag management capabilities. This is installed as a standalone uber jar from the 
-[Nessie download page](../downloads/index.md) or the
-[releases page on GitHub](https://github.com/projectnessie/nessie/releases/). 
+and tag management capabilities. 
 
 Nessie CLI is designed to be usable as an interactive REPL supporting auto-completion,
 highlighting where appropriate and has built-in help. Long outputs, like a commit log,
 are automatically paged like the Unix `less` command.
 
+![Nessie CLI](../img/cli-intro.png)
 
+## Installation
 
-![Nessie CLI](../img/cli-intro.png)
+Nessie CLI can be installed in two different ways:
```

Comment:
Linking the downloads page is good enough. Can't reference the "right" version here (see my comment below)